Anima/diffusion/mcm/animaMultiCompartmentModelCreator.cxx

```diff
@@ -69,17 +57,18 @@ MultiCompartmentModelCreator::MCMPointer MultiCompartmentModelCreator::GetNewMul
     outputMCM->SetCommonExtraAxonalFractionParameters(m_UseCommonExtraAxonalFractions);
 
     double sumIsotropicWeights = 0;
+    double isoDefaultWeight = 0.1;
```

Comment:
Ok I might need some explanation about this `isoDefaultWeight`. Didn't you remove the fixed weight possibility?

Reply:
The fixed weights option is removed. But there is not only varpro in life and so we need to set the isotropic compartments some weight with respect to other compartments. I kept that to have anisotropic compartments get larger weights by default as an initialization. But we could go for having all compartments having indifferently all the same weight when starting the optimization. Will do this.